### PR TITLE
Add To Launcher Button for Stereum Servers

### PR DESCRIPTION
**Pull request by MaxTheGeeek**
### Description
Add "Add To Launcher" button that enables users to import purchased servers from the web app directly into the Stereum Launcher with a single click.

### Technical Implementation
- Implemented custom URL protocol handler (`stereumlauncher://`)
- Added protocol handling for all platforms (Windows, macOS, Linux)
- Created server data import functionality
- Added server validation and duplicate checking

### Features
- [x] One-click server import from web to desktop app
- [x] Automatic launcher opening
- [x] Server data validation
- [x] Duplicate server detection
- [x] Cross-platform support

### Testing Status
- [x] macOS: Tested and working
- [x] Windows: Tested and working
- [x] Linux: Tested and working

### CODE REVIEW

**Comment by MaxTheGeeek** [comment]
Thoroughly tested and confirmed to work seamlessly on macOS, Windows, and Linux. It successfully opens the application and handles protocol data as expected across all platforms 🔥🚀😊

File: launcher/src/background.js [hunk 1]
```diff
@@ -1023,30 +1019,62 @@ app.on("web-contents-created", (event, contents) => {
...
+  await createWindow();
+});
 
-  // Disable "View" and "Window" Menu items in build (since CTRL+R and F5 is disabled also)
-  if (!isDevelopment) {
-    const hideMenuItems = ["viewmenu", "windowmenu"];
-    var menu = Menu.getApplicationMenu();
-    menu.items.filter((item) => hideMenuItems.includes(item.role)).map((item) => (item.visible = false));
-    Menu.setApplicationMenu(menu);
-    stereumUpdater.checkForUpdates();
```
Comment (NeoPlays) [inline launcher/src/background.js:1041]: i think you remeoved `stereumUpdater.checkForUpdates()`

**Review by NeoPlays — CHANGES_REQUESTED** [review]
lgtm otherwise 🚀

File: launcher/src/background.js (hunk 1 above)
Comment (MaxTheGeeek) [reply launcher/src/background.js:1041]: seems, u r right.

**Review by NeoPlays — APPROVED** [review]

**Pull request merged**